cmd/dp-observation-importer/main.go

```diff
@@ -82,69 +129,122 @@ func main() {
 	observationStore := observation.NewStore(dimensionIDCache, graphDB, errorReporter)
 
 	// write import results to kafka topic.
-	resultWriter := observation.NewResultWriter(kafkaResultProducer)
+	resultWriter := observation.NewResultWriter(observationsImportedProducer)
 
 	// handle a batch of events.
 	batchHandler := event.NewBatchHandler(observationMapper, observationStore, resultWriter, errorReporter)
 
 	eventConsumer := event.NewConsumer()
 
 	// Start listening for event messages.
-	eventConsumer.Consume(kafkaConsumer, cfg.BatchSize, batchHandler, cfg.BatchWaitTime, errorChannel)
-
-	signals := make(chan os.Signal, 1)
-	signal.Notify(signals, os.Interrupt, syscall.SIGTERM)
+	eventConsumer.Consume(ctx, syncConsumerGroup, cfg.BatchSize, batchHandler, cfg.BatchWaitTime, errorChannel)
 
+	// block until a fatal error occurs
```

Comment:
We removed the Error cases from the `select` because we don't want to kill the app in that case, but we still want to log the errors. I would suggest adding the dp-kafka logging go routines: 
```
syncConsumerGroup.Channels().LogErrors(ctx, "Consumer error")
observationsImportedProducer.Channels().LogErrors(ctx, "Observations Imported Producer error")
observationsImportedErrProducer.Channels().LogErrors(ctx, "Observation Imported Error Producer error")

```